journal/vendor-extra/PatternsBundle/Resources/views/views.html.twig

```diff
@@ -0,0 +1,10 @@
+{% for view in views -%}
+
+    {% if view.template is defined %}
+        {%- include view.template with view.arguments only -%}
```

Comment:
This allows embedding arbitrary templates. Avoids the annoyance with eLife's Mustache templates where you have to render things early then include them as strings.